tests/TestHelpers/GraphHelper.php

```diff
@@ -35,23 +35,10 @@ public static function getUUIDv4Regex(): string {
 	}
 
 	/**
-	 * @param string $id
-	 *
-	 * @return bool
-	 */
-	public static function isUUIDv4(string $id): bool {
-		$regex = "/^" . self::getUUIDv4Regex() . "$/i";
-		return (bool)preg_match($regex, $id);
-	}
```

Comment:
don't we use this function in other place?

Reply:
previously used but those functions has been cleaned up as of now.